src/MetadataUtility/Audio/Vendors/FrontierLabs.cs

```diff
@@ -5,27 +5,42 @@
 namespace MetadataUtility.Audio.Vendors
 {
     using System.Buffers.Binary;
+    using System.Diagnostics;
+    using System.Reflection;
     using System.Text;
     using LanguageExt;
-    using LanguageExt.Common;
```

Comment:
Ok, Models shouldn't be imported into this class so that's one fix.

But given the use of LanguageExt in this class I'd guess it'd be more prevalent than the models

Reply:
Makes sense, won't be needed in here once I rearchitect.